// @expo-google-fonts/cormorant/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Cormorant_300Light = require('./300Light/Cormorant_300Light.ttf');
export const Cormorant_400Regular = require('./400Regular/Cormorant_400Regular.ttf');
export const Cormorant_500Medium = require('./500Medium/Cormorant_500Medium.ttf');
export const Cormorant_600SemiBold = require('./600SemiBold/Cormorant_600SemiBold.ttf');
export const Cormorant_700Bold = require('./700Bold/Cormorant_700Bold.ttf');
export const Cormorant_300Light_Italic = require('./300Light_Italic/Cormorant_300Light_Italic.ttf');
export const Cormorant_400Regular_Italic = require('./400Regular_Italic/Cormorant_400Regular_Italic.ttf');
export const Cormorant_500Medium_Italic = require('./500Medium_Italic/Cormorant_500Medium_Italic.ttf');
export const Cormorant_600SemiBold_Italic = require('./600SemiBold_Italic/Cormorant_600SemiBold_Italic.ttf');
export const Cormorant_700Bold_Italic = require('./700Bold_Italic/Cormorant_700Bold_Italic.ttf');
