// @expo-google-fonts/archivo/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Archivo_100Thin = require('./100Thin/Archivo_100Thin.ttf');
export const Archivo_200ExtraLight = require('./200ExtraLight/Archivo_200ExtraLight.ttf');
export const Archivo_300Light = require('./300Light/Archivo_300Light.ttf');
export const Archivo_400Regular = require('./400Regular/Archivo_400Regular.ttf');
export const Archivo_500Medium = require('./500Medium/Archivo_500Medium.ttf');
export const Archivo_600SemiBold = require('./600SemiBold/Archivo_600SemiBold.ttf');
export const Archivo_700Bold = require('./700Bold/Archivo_700Bold.ttf');
export const Archivo_800ExtraBold = require('./800ExtraBold/Archivo_800ExtraBold.ttf');
export const Archivo_900Black = require('./900Black/Archivo_900Black.ttf');
export const Archivo_100Thin_Italic = require('./100Thin_Italic/Archivo_100Thin_Italic.ttf');
export const Archivo_200ExtraLight_Italic = require('./200ExtraLight_Italic/Archivo_200ExtraLight_Italic.ttf');
export const Archivo_300Light_Italic = require('./300Light_Italic/Archivo_300Light_Italic.ttf');
export const Archivo_400Regular_Italic = require('./400Regular_Italic/Archivo_400Regular_Italic.ttf');
export const Archivo_500Medium_Italic = require('./500Medium_Italic/Archivo_500Medium_Italic.ttf');
export const Archivo_600SemiBold_Italic = require('./600SemiBold_Italic/Archivo_600SemiBold_Italic.ttf');
export const Archivo_700Bold_Italic = require('./700Bold_Italic/Archivo_700Bold_Italic.ttf');
export const Archivo_800ExtraBold_Italic = require('./800ExtraBold_Italic/Archivo_800ExtraBold_Italic.ttf');
export const Archivo_900Black_Italic = require('./900Black_Italic/Archivo_900Black_Italic.ttf');
